datadog_checks_base/datadog_checks/base/checks/win/wmi/__init__.py

```diff
@@ -84,16 +85,20 @@ def _raise_on_invalid_tag_query_result(self, sampler, wmi_obj, tag_query):
                 message = "multiple results returned (one expected)"
 
             self.log.warning(
-                u"Failed to extract a tag from `tag_queries` parameter: {reason}."
-                " wmi_object={wmi_obj} - query={tag_query}".format(reason=message, wmi_obj=wmi_obj, tag_query=tag_query)
+                u"Failed to extract a tag from `tag_queries` parameter: %s. wmi_object=%s - query=%s",
+                message,
+                wmi_obj,
+                tag_query,
             )
             raise TagQueryUniquenessFailure
 
         if sampler[0][target_property] is None:
+            wmi_property = target_property
+            wmi_class = target_class
```

Comment:
Any reason to create new variables here?

Reply:
Nice catch!